### DISCUSSION TITLE
Case for Megadesk Companion

### DISCUSSION THREAD

**Discussion by madnat** [gcormier/megadesk#152] [General]
Has anyone created a 3D printable case for the companion Board?  Please share if you have.  If Not - @gcormier could you put pcb files for Companion so I can create one to fit the board and the hole/usb-C connector.

Thanks

**Comment by gcormier** [comment]
Cases - https://github.com/gcormier/megadesk_companion/tree/main/case

All the PCB models (STEP) are in the assets of each release - https://github.com/gcormier/megadesk_companion/releases

**Comment by madnat** [comment]
Ahh - didn’t fully appreciate there was a separate repository for the companion board - makes sense.

case printed - thanks.
